src/GafferScene/Instancer.cpp

```diff
@@ -2808,9 +2808,10 @@ void Instancer::InstancerCapsule::render( IECoreScenePreview::Renderer *renderer
 		}
 	}
 
-
-
-	tbb::parallel_for( tbb::blocked_range<size_t>( 0, instancesForPrototype.size() ), [&]( const tbb::blocked_range<size_t> &r )
+	// We've found problems with performance when running too many iterations in parallel, so we set the grain
+	// size so that we shouldn't use more than 32 threads
```

Comment:
We need to be specific about how and why we're tuning this - `AiNode()` bottlenecks.